Sources/Brave/Frontend/Sync/BraveCore/BraveSyncAPIExtensions.swift

```diff
@@ -55,9 +55,9 @@ extension BraveSyncAPI {
   }
   
   @discardableResult
-  func joinSyncGroup(codeWords: String, syncProfileService: BraveSyncProfileServiceIOS) -> Bool {
+  func joinSyncGroup(codeWords: String, syncProfileService: BraveSyncProfileServiceIOS, shouldEnableBookmarks: Bool = true) -> Bool {
```

Comment:
I would avoid default parameters just because we don't want someone to accidentally use this function and "accidentally" enable bookmarks syncing when they shouldn't.

Reply:
You are right , this actually showed me a an error in the PR. Fix at https://github.com/brave/brave-ios/pull/7918/commits/018fa5da8b358c050d1d9f7d00e454b06f316a75